deployments/demos/local-containers/README.md

```diff
@@ -0,0 +1,209 @@
+
+```mermaid
+graph TD
+    %% --- Style Definitions ---
+    classDef machine fill:#f9f9f9,stroke:#333,stroke-width:2px,stroke-dasharray:5 5
+    classDef network fill:#e6f3ff,stroke:#0066cc,stroke-width:1.5px
+    classDef service fill:#ffffff,stroke:#333
+    classDef bridge fill:#fffbe6,stroke:#f0ad4e
+    classDef invisible stroke:none,fill:none
+
+    %% === Local Client Machine ===
+    subgraph LocalClient[Local Client Machine]
+        direction LR
+        class LocalClient machine
+
+        subgraph VisualizerNet[visualizer_net Network]
+            direction TB
+            class VisualizerNet network
+            
+            visualizer["**Visualizer**<br><br>Based on RViZ2<br>Provides noVNC Remote Desktop"]
+            visualizer_bridge["**Visualizer Zenoh Bridge**<br><br><u>Role</u>: Client<br>Converts Zenoh ↔️ DDS"]
+            class visualizer service
+            class visualizer_bridge bridge
+            
+            visualizer -->|"ROS2 DDS Data"| visualizer_bridge
+        end
+    end
+
+    %% === Remote Server ===
+    subgraph RemoteServer[Remote Server]
+        direction LR
+        class RemoteServer machine
+
+        subgraph AutowareNet[autoware_net Network]
+            direction TB
+            class AutowareNet network
+
+            autoware["**Autoware**<br><br>Core Autonomous Driving Algorithms<br>Perception, Planning, Control"]
+            scenario_simulator["**Scenario Simulator**<br><br>Provides Simulation Scenarios<br>With Sensor Data"]
+            autoware_bridge["**Autoware Zenoh Bridge**<br><br><u>Role</u>: Router<br>Listens on TCP/7447<br>Converts DDS ↔️ Zenoh"]
+            class autoware,scenario_simulator service
+            class autoware_bridge bridge
+
+            autoware <-->|"ROS2 DDS Data"| scenario_simulator
+            autoware -->|"ROS2 DDS Data"| autoware_bridge
+            scenario_simulator -->|"ROS2 DDS Data"| autoware_bridge
+        end
+    end
+
+    %% === External & Cross-Network Connections ===
+    user[fa:fa-user User] -->|"HTTP (Port 6080)"| visualizer
+    visualizer_bridge <-->|"<b>Zenoh Protocol over zenoh_net</b><br>Connects to tcp/autoware_zenoh_bridge:7447"| autoware_bridge
+```
+
+### 2.3. Network Isolation and Communication Bridge
+
+- **Network Design**:
+  - `autoware_net`: An isolated virtual network for `autoware` and `scenario_simulator`, using ROS 2 DDS multicast for low-latency communication.
+  - `visualizer_net`: An isolated network for `visualizer`, simulating physical or logical separation from the server.
+  - `zenoh_net`: A dedicated bridge network connecting only `autoware_zenoh_bridge` and `visualizer_zenoh_bridge`, ensuring a clean cross-domain data transmission path.
+
+- **Communication Core: Zenoh Bridge**:
+  - `autoware_zenoh_bridge`: Acts as a **Router**, scanning ROS 2 topics in `autoware_net`, converting them to Zenoh format, and listening for client connections on TCP/7447 in `zenoh_net`.
```

Comment:
I think we can mention both `autoware_zenoh_bridge` and `visualizer_zenoh_bridge` are `zenoh-bridge-ros2dds` to avoid confusion.